### DISCUSSION TITLE
Send Group Message: blocked by server

### DISCUSSION THREAD

**Discussion by abdofallah** [howmanybots/onebot#240] [问答]
When I try to use:
`await bot.send_group_msg(group_id=event.group_id, message=MessageSegment.text(f'\n\nHow can I help you?'), auto_escape=True)`
i get an error:
`send group message failed: blocked by server', msg='SEND_MSG_API_ERROR', retcode=100, status='failed', wording='send group message failed: blocked by server') `

but if i add MessageSegment.at() or MessageSegment.reply() in the message too it works just fine.

What if I do not want to mention/or reply to the message?

**Comment by stdrc** [comment]
I guess this issue is with https://github.com/Mrs4s/go-cqhttp (if you are indeed using it), you may want to raise an issue/discussion there.
